src/views/Home/components/Speakers/speakers.scss

```diff
@@ -1,3 +1,29 @@
-/* stylelint-disable-line no-empty-source */
+.Speakers {
+  margin: auto;
+  position: relative;
 
-/* Delete these lines when you begin development */
+  &__titleContainer {
+    display: flex;
+    width: 15em;
+    height: 2.5em;
+    background: #fdfdee;
+    border-radius: 64px;
+    margin: auto;
+    justify-content: center;
+    align-items: center;
+  }
+
+  &__title {
+    color: var(--salmon);
+  }
+
+  &__cardsContainer {
+    max-width: 1800px;
+    min-height: 50em;
+    margin: auto;
+    display: flex;
+    flex-wrap: wrap;
+    align-content: space-around;
+    align-items: center;
```

Comment:
Because each speaker card is strictly bound by its padding and margins, I'm not sure how much of this is needed too.